config/initializers/rack_timeout.rb

```diff
@@ -2,20 +2,37 @@
 
 module Rack
   class Timeout
-    @excludes = [
+    EXCLUDES = [
+      # API routes do not have path-based locale parameters
       '/api/verify/images',
-      '/verify/doc_auth/document_capture',
-      '/verify/doc_auth/verify',
-      '/verify/capture_doc/document_capture',
-      '/verify/doc_auth/link_sent',
+
+      '/verify/verify_info',
+      '/en/verify/verify_info',
+      '/es/verify/verify_info',
+      '/fr/verify/verify_info',
```

Comment:
how would you feel about building the locale-specific URLs more dynamically? so more like:

```
EXCLUDES = [
  '/verify/verify_info'
].flat_map do |path|
  [path] + I18n.available_locales.map { |locale| "/#{locale}/#{path}" }
end
```

Reply:
This runs into an issue where `I18n.available_locales` is not set correctly yet since it is running before the application has initialized.

Reply:
what if we moved `    config.i18n.available_locales = %w[en es fr]` into a separate initializer file, and made sure it runs before this one by renaming the files, ex `config/initializers/00_i18n.rb`

Reply:
or we could move that list of locales to a constant (say in `lib`) that we can require from both places

Reply:
I took the second option in 9a46bf0238